WordPress/Classes/ViewRelated/Suggestions/SuggestionsTableViewCell.m

```diff
@@ -58,14 +58,18 @@ - (void)setupConstraints
                             @"title": _titleLabel,
                             @"subtitle": _subtitleLabel,
                             @"icon": _iconImageView };
-        
-    NSDictionary *metrics = @{@"iconsize": @(SuggestionsTableViewCellIconSize) };
-        
+
     // Horizontal spacing
-    NSArray *horizConstraints = [NSLayoutConstraint constraintsWithVisualFormat:@"H:|-[icon(iconsize)]-16-[title]-[subtitle]-|"
+    NSArray *horizConstraints = [NSLayoutConstraint constraintsWithVisualFormat:@"H:|-[icon]-16-[title]-[subtitle]-|"
                                                                         options:0
-                                                                        metrics:metrics
+                                                                        metrics:nil
                                                                           views:views];
+
+    [NSLayoutConstraint activateConstraints:@[
```

Comment:
We can no longer gurantee that the images are resized to exactly match the view size.